### ISSUE TITLE
Scrub TODOs and comments referencing closed bugs.

### ISSUE THREAD

**Issue by benshayden** [catapult-project/catapult#3285]
I noticed a few comments referencing closed bugs that should either be done because they're unblocked, or clarified that the links are for historical purposes, or removed because we have git blame.

 * [find_input_expectations](https://github.com/catapult-project/catapult/blob/4ffbbd740839d9a2030e931d8e190a95ae7f51d7/tracing/tracing/importer/find_input_expectations.html#L820)
 * [drawing_container_perf_test](https://github.com/catapult-project/catapult/blob/ff40d4569c8bcf2eed27ed49682e7b8198d24dc1/tracing/tracing/ui/tracks/drawing_container_perf_test.html#L87)
 * [user_model_builder](https://github.com/catapult-project/catapult/blob/529a4d455fb4133b776678b003a834a45e7432c8/tracing/tracing/importer/user_model_builder.html#L46)
 * [tracing_metric](https://github.com/catapult-project/catapult/blob/5e87e6be4f5dea35566a79d1b75a627cb4d83876/tracing/tracing/metrics/tracing_metric.html#L108)
 * [v8/execution_metric](https://github.com/catapult-project/catapult/blob/476d031f85fa622d1688ba2e3c08b2a5bb8d437a/tracing/tracing/metrics/v8/execution_metric.html#L134)

That is not an exhaustive list, and I only looked in tracing/tracing.
It's mostly (but not only) my code. :-)

This is low priority, but if we ever have the breathing room to scrub code health bugs, this could go somewhere on the list.

This also points to a bug hygiene issue: those bugs probably shouldn't have been closed until that code was updated. That could lead to a philosophical discussion on TODOs: should they be discouraged in favor of filing and using bugs? Etc.

@eakuefner @zeptonaut

**Comment by zeptonaut** [comment]
I like TODOs, but I totally agree that our current method of handling them and associating them with bugs sucks.

It seems like the problem here is, more than anything, a tooling problem. On my old team, people regularly turned on the [dead code layer of Code Search](https://cs.corp.google.com/piper///depot/google3/java/com/google/ads/servers/gfp/?q=f:forecastimporter&sq=package:piper+file://depot/google3+-file:google3/experimental&dr) in order to find code that could be deleted. Code Search also has a "stale TODO" layer that, for `TODO(b/12345)`, identifies whether buganizer bug 12345 is closed.

Interestingly, it looks like the [Catapult code search](https://cs.corp.google.com/github/catapult-project/catapult/tracing/tracing/model/) has these same layers, but it doesn't look like the indexer runs on them. It'd be really awesome to get something like that turned on, but I'm not sure how practical it would be: a simple script that finds Catapult and monorail bug links and uses the monorail API to determine if those bugs are closed would probably be sufficient. Then, it can be a sort of best practice for our team to occasionally go through and scrub the codebase of obsolete bugs.
